packages/core/src/actions/public/get-ibc-client-query-client.ts

```diff
@@ -0,0 +1,18 @@
+import { CosmWasmClient } from '@cosmjs/cosmwasm-stargate'
+
+import {
+  AnsHostQueryClient,
+  IbcClientQueryClient,
+} from '../../codegen/abstract'
+import { WithArgs } from '../../types/with-args'
+
+export type GetIbcClientQueryClientParameters = WithArgs<{
+  cosmWasmClient: CosmWasmClient
+  ibcClientAddress: string
+}>
+
+export async function getIbcClientQueryClient({
```

Comment:
doesn't need to be async
```suggestion
export function getIbcClientQueryClient({
```